_posts/2015-10-07-code-talks-2015-retrospective.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "code.talks 2015 retrospective"
+date: "2015-10-07 09:00:00"
+icon: comments
+categories: conferences
+authors: ["Jens"]
+---
+
+For the fifth time about 1.500 developers gathered for the yearly `code.talks` conference in Hamburg:
+on September 29 and 30 also a dozen ePagees went to the Cinemaxx Hamburg Dammtor to enjoy two days fully packed with [lots of talks](https://www.codetalks.de/2015/programm) from 13 different tracks.
+Each track focussed on a specific theme like E-Commerce, IT Management, Startups, New Technology, Big Data, Mobile, DevOps, Scaling, Infrastructure, UX/Frontend, PHP, JavaScript and Java - the latter also being sponsored by ePages.
+Each hour talks from these tracks where held in parallel at eight movie theater rooms.
```

Comment:
Some explanation: talks started at each full hour and went for 45 minutes followed by 15 minutes break. Lunch break was from 12:00 until 13:00. So I try this wording:

"Talks from these tracks where held in parallel at eight movie theater rooms. 
Each talk was 45 minutes long, with a small break afterwards plus a longer break for lunch."